setup.cfg

```
@@ -39,7 +39,9 @@ zip_safe = false
 tests =
     pytest
     pytest-asyncio
-    mypy>=0.800
+    # Pending release of https://github.com/python/mypy/pull/17355
+    # Likely in v1.13.1+, when pip can be removed.
+    mypy~=1.11.0
```

Checks pass against mypy's `master` branch with: 

```suggestion
    mypy @ git+https://github.com/python/mypy.git@master
```